packages/fetchai/connections/ledger_api/base.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
```

Comment:
I refactored the `LedgerApiConnection`. Now it has two different request dispatcher: one for the ledger api protocol and the other for the contract api protocol. This file contains the abstract request handler.